src/classLibrary/EvaluationStage/CyanParser.ts

```diff
@@ -0,0 +1,35 @@
+import { CyanObject, CyanSafe, ICyanParser, Syntax } from "../interfaces/interfaces";
+import _ from "lodash";
+
+class CyanParser implements ICyanParser {
+
+    Parse(cyanObject: Partial<CyanObject>): CyanSafe {
+        let defaultCyanSafe: CyanSafe = {
+            globs: [],
+            copyOnly: [],
+            variable: {},
+            flags: {},
+            guid: [],
+            syntax: [],
+            plugins: {},
+            comments: [],
+            pluginData: {}
+        }
+
+        if (cyanObject.globs) defaultCyanSafe.globs = _.concat([], cyanObject.globs);
+        if (cyanObject.copyOnly) defaultCyanSafe.copyOnly = _.concat([], cyanObject.copyOnly);
+        if (cyanObject.variable) defaultCyanSafe.variable = cyanObject.variable;
+        if (cyanObject.flags) defaultCyanSafe.flags = cyanObject.flags;
+        if (cyanObject.guid) defaultCyanSafe.guid = _.concat([], cyanObject.guid);
+        if (cyanObject.syntax) {
+            defaultCyanSafe.syntax = _.chunk(_.flattenDeep(cyanObject.syntax), 2) as Syntax[];
```

Comment:
do you need chunk? Also es2019 + supports .flat(). Try cut down dependency on lodash as much as possible

Reply:
Yeah we need chunk to make sure that the syntax is paired up again. This is because a Syntax is really just [string, string]